docs/ReleaseNotes.md

```diff
@@ -1,6 +1,22 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.3.2
+Acuminator 2.3.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.3.2, the following enhancements have been implemented:
+ - The [PX1076](diagnostics/PX1076.md) diagnostic has been added to check for and warn a developer that the PXInternalUseOnly attribute is used in the customization code.
+ - The documentation for the [PX1046](diagnostics/PX1046.md) diagnostic was extended, the reasoning for the diagnostic is now provided.
+ 
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+ - Due to the changes in the theme updating mechanism, an error was thrown by Visual Studio in the Acuminator colorizer, and the Acuminator coloring became corrupted.
+ - Due to the changes in the latest Visual Studio updates, Acuminator custom settings were not enabled, and the Acuminator analyzers used default values.
+ - In the [PX1036](diagnostics/PX1036.md) diagnostic, foreign keys for different DACs that use the same set of DAC fields were incorrectly considered as duplicates.
+ - Due to the changes in the latest Visual Studio updates, switching between themes was impossible. 
```

Comment:
I think the VS version should be specified:
`The change of a current Visual Studio theme didn't work correctly for Acuminator coloring in Visual Studio 2019`